doc/releases/changelog-dev.md

```diff
@@ -52,6 +52,71 @@
 
 <h4>Catalyst is seamlessly integrated with PennyLane ⚗️</h4>
 
+* The ``qml.compiler`` module provides support for hybrid quantum-classical compilation.
+  [(#4692)](https://github.com/PennyLaneAI/pennylane/pull/4692)
+
+  Through the use of the ``qml.qjit`` decorator, entire workflows can be just-in-time (JIT)
```

Comment:
We can use single backticks because it's markdown

Reply:
```suggestion
  Through the use of the `qml.qjit` decorator, entire workflows can be just-in-time (JIT)
```